style/dpms.css

```diff
@@ -53,14 +51,13 @@
     cursor: pointer;
   }
   
-  div[role="treeitem"][aria-level="2"] > div {
+ div[role="treeitem"][aria-level="2"] > div {
     display: flex;
     margin-left: 20px;
-  }
-  
+  }   
   div[role="treeitem"][aria-level="2"] > div > div > svg {
```

Comment:
ditto

Reply:
Added a class along with this. The tree library used doesn't giving any specific class and so we have used like this.